azurerm/config.go

```diff
@@ -233,6 +235,7 @@ func getArmClient(c *authentication.Config, skipProviderRegistration bool, partn
 	client.frontdoor = frontdoor.BuildClient(o)
 	client.graph = graph.BuildClient(o)
 	client.hdinsight = hdinsight.BuildClient(o)
+	client.healthcare = healthcare.BuildClient(o)
```

Comment:
since this is a new Resource Provider can we also add `` to the [list of Required Resource Providers](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/required_resource_providers.go#L18) that the Azure Provider attempts to register automatically?

This allows us to provide better error messages to users

Reply:
added "Microsoft.Healthcare" to the list

Reply:
added "Microsoft. Healthcare" to the list